efficient_full_table_scan_example_code/README.md

```diff
@@ -89,3 +89,54 @@ Done!
 
 Total Execution Time: 15m32s
 ```
+
+
+
+Generic GO installation and script setup into ~/gocqlcount
+====================================================
+
+E.g. for go 1.20.4:
+
+```
+cd ~
+
+GOVERSION=go1.20.4
+curl -o $GOVERSION.linux-amd64.tar.gz https://dl.google.com/go/$GOVERSION.linux-amd64.tar.gz
+
+tar xzvf $GOVERSION.linux-amd64.tar.gz
+
+ln -s $HOME/go $HOME/$GOVERSION 
```

Comment:
why not install it globally?

Reply:
to not break global tools? 
installing globally is for package manager, not really for inplace script as this
I can add a pointer to installer of go for various OSes ... oh wait, it's not packaged
you always manage it like: https://go.dev/doc/manage-install ... d'oh
OK, I can rewrite above to global then or point to go docs

Reply:
I would just point to official install guide. This script isn't maintenable

Reply:
@tarzanek ping

Reply:
resolved, pointed to official guide, left the one as sample and fixed per Macieks comments